docs/source/markdown/podman-pod-create.1.md.in

```diff
@@ -21,10 +21,10 @@ Name (“jonah”)
 
 podman generates a UUID for each pod, and if a name is not assigned
 to the container with **--name** then a random string name will be generated
-for it. The name is useful any place you need to identify a pod.
+for it. This name is useful to identify a pod.
 
 Note: resource limit related flags work by setting the limits explicitly in the pod's cgroup
-which by default, is the cgroup parent for all containers joining the pod. Containers are still delegated the ability to set their own resource limits when joining a pod meaning that if you run **podman pod create --cpus=5** you can also run **podman container create --pod=`<pod_id|pod_name>` --cpus=4** and the container will only see the smaller limit. containers do NOT get the pod level cgroup resources if they specify their own cgroup when joining a pod such as **--cgroupns=host**
+which by default, is the cgroup parent for all containers joining the pod. Containers are still delegated the ability to set their own resource limits when joining a pod meaning that if the pod was created via **podman pod create --cpus=5** specifying **podman container create --pod=`<pod_id|pod_name>` --cpus=4** causes the container see the smaller limit. containers do NOT get the pod level cgroup resources if their specified with their own cgroup when joining a pod such as **--cgroupns=host**
```

Comment:
This one makes my head hurt but I can't find a way to rephrase it. So, just two minor notes:
* comma wanted here: "when joining a pod >>,<< meaning ..."
* "Containers" needs to be capitalized in the last sentence
* **they're**, not their
* I can't count to two any more